src/styles/main.scss

```diff
@@ -1,3 +1,9 @@
+@import "./utils/variables.scss";
+@import "./blocks/grid.scss";
+@import "./blocks/cards.scss";
+@import "./blocks/header.scss";
+@import "./blocks/stars.scss";
+
 body {
   margin: 0;
```

Comment:
```suggestion
  margin: 0;
  font-family: "Roboto", sans-serif;
```

Reply:
Consider applying font styles for the body tag to get them inherited by other elements

Reply:
Also, it's better to create a separate file for reset styling to keep the main file clean and clear